api/ocm/extensions/repositories/genericocireg/repository.go

```diff
@@ -44,23 +48,70 @@ type RepositoryImpl struct {
 	nonref   cpi.Repository
 	ocirepo  oci.Repository
 	readonly bool
+	// blobLimit is the size limit for layers maintained for the storage of localBlobs.
+	// The value -1 means an unconfigured value,
+	// a value == 0 disables the limiting and
+	// a value > 0 enabled the usage of the specified size.
```

Comment:
So:
-1: if a blob limit configuration is found for the host:port, it's used
== 0: even if a blob limit configuration is found for the host:port, it's ignored
> 0: if a blob limit configuration is found for the host:port, it won't be applied but the concrete value assigned here will be used

that right?

Reply:
clarified